src/Util/loader.php

```diff
@@ -73,7 +73,18 @@ function get_muplugins(string $abs = ABSPATH, string $pdir = WP_PLUGIN_DIR, stri
         }
         set_site_transient($key, $plugins);
     }
-    return $plugins;
+
+    /**
+     * Filter the list of Must-Use plugins. This must be done very early, likely
+     * by creating a file in the `mu-plugins` directory which is alphabetically
+     * before this MU-Loader is loaded. 
+     * 
+     * @param string[] $plugins The list of Must-Use plugin files, relative to
+     *                         the Must-Use plugins directory.
+     * 
+     * @return string[] The filtered list of Must-Use plugins.
+     */
+    return (array) apply_filters( 'lkwdwrd_mupluginloader_get_muplugins', $plugins );
```

Comment:
Can this have an easier to remember name? I understand why it's called that, but does require remembering the username of the author of this package.

Worth adding documentation too.

Reply:
Having looked through the rest of the file, it does make sense to keep the namespacing consistent.

But still worth documenting the addition.

Reply:
Yeah, initially I called it something else, then saw _all_ the other actions filters and decided that it was better to be consistent.